File: others/go/fibonacci.go
package main

import "fmt"

func main() {
  fmt.Println(fibo(13))
}

func fibo(n int) int {
  if n == 0 {
    return 0
  }
  a := 0
  b := 1
  var c int

  for i := 0; i < n; i++ {
    c = a + b
    a = b
    b = c
  }
  return b
}

File: others/julia/fibonacci.jl
fibonacci(n::Int) = begin
  fibo(n::Int, a::Int, b::Int) = begin
    n == 0 && return a
    fibo(n - 1, b, a + b)
  end

  fibo(n, 0, 1)
end

map(fibonacci, 0:10) |> println
